moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/jog_interface_base.h

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
I'd prefer the license headers to match the format of the other sources in the repo.

Reply:
Can I do this in a follow-up PR? We just merged an update to the license formatting that's going to conflict -- https://github.com/ros-planning/moveit/pull/1764

Reply:
done!